ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -136,10 +142,24 @@ const ConfirmAddSuggestedToken = () => {
         });
       }),
     );
-
     history.push(mostRecentOverviewPage);
   }, [dispatch, history, trackEvent, mostRecentOverviewPage, suggestedAssets]);
 
+  const handleCancelClick = useCallback(async () => {
+    await Promise.all(
+      suggestedAssets.flatMap(({ id }) => [
+        dispatch(
+          rejectPendingApproval(
+            id,
+            serializeError(ethErrors.provider.userRejectedRequest()),
```

Comment:
Copied this error from the permissions confirmation screen.

https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/permissions-connect/permissions-connect.component.js#L383